stubs/Modifiers/IndexFilesModifier.swift

```diff
@@ -33,8 +33,7 @@ class IndexFileModifier {
         let removeQueryKeys: [String] = modificationRulesObject["remove"] as? [String] ?? []
 
         if addDictionary.isEmpty && removeQueryKeys.isEmpty {
-            printError("Need to have at least one modification rule in object\n\(modificationRulesObject)")
-            abort()
+            fatalError("Need to have at least one modification rule in object\n\(modificationRulesObject)")
```

Comment:
I was having major issues trying to debug an issue. I thought using `fatalError` may have worked better than `abort`. It turns out it made no difference. That being said, I'm keeping `fatalError` because 1. It takes only one line to represent the idea 2. `printError` was turning my entire console red. I could have fixed #2 or even created a third option to display an error and then quit... maybe later?

Reply:
I am ok with this change, lets leave it for now